<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>683</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">  QWidget {
        background-color: #333333;
        color: #ffffff;
        border: none;
    }
    QPushButton {
        background-color: #4d4d4d;
        border: 1px solid #4d4d4d;
        border-radius: 4px;
        color: #ffffff;
        padding: 5px;
    }
    QPushButton:hover {
        background-color: #5a5a5a;
        border: 1px solid #5a5a5a;
    }
    QCheckBox {
        color: #ffffff;
    }
    QLineEdit {
        background-color: #4d4d4d;
        border: 1px solid #4d4d4d;
        color: #ffffff;
        padding: 5px;
    }
    QTextEdit {
        background-color: #4d4d4d;
        border: 1px solid #4d4d4d;
        color: #ffffff;
        padding: 5px;
    }
    QProgressBar {
        border: 1px solid #444444;
        border-radius: 7px;
        background-color: #2e2e2e;
        text-align: center;
        font-size: 10pt;
        color: white;
    }
    QProgressBar::chunk {
        background-color: #3a3a3a;
        width: 5px;
    }
    QScrollBar:vertical {
        border: none;
        background-color: #3a3a3a;
        width: 10px;
        margin: 16px 0 16px 0;
    }
    QScrollBar::handle:vertical {
        background-color: #444444;
        border-radius: 5px;
    }
    QScrollBar:horizontal {
        border: none;
        background-color: #3a3a3a;
        height: 10px;
        margin: 0px 16px 0 16px;
    }
    QScrollBar::handle:horizontal {
        background-color: #444444;
        border-radius: 5px;
    }
    QTabWidget {
        background-color: #2e2e2e;
        border: none;
    }
    QTabBar::tab {
        background-color: #2e2e2e;
        color: #b1b1b1;
        padding: 8px 20px;
        border-top-left-radius: 5px;
        border-top-right-radius: 5px;
        border: none;
    }

    QTabBar::tab:selected, QTabBar::tab:hover {
        background-color: #3a3a3a;
        color: white;
    }</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="topMargin">
        <number>50</number>
       </property>
       <item>
        <widget class="QLabel" name="unload">
         <property name="styleSheet">
          <string notr="true">font: 700 56pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Select Spreadsheet To Unload</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>350</number>
       </property>
       <item>
        <widget class="QComboBox" name="unload_sheets">
         <property name="minimumSize">
          <size>
           <width>700</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>1050</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QComboBox {
    /* Main styling for the combo box */
    background-color: #f0f0f0; /* Background color */
    border: 1px solid #999; /* Border color */
    padding: 5px; /* Padding inside the combo box */
	color: #333333;
    font-size: 20px; /* Replace with your desired font size */
    font-weight: bold; /* Make the text bold */



    /* Dropdown button styling */
    QComboBox::drop-down {
        background-color: #ddd; /* Dropdown button background color */
        border: 1px solid #999; /* Dropdown button border color */
    }

}

QComboBox::drop-down:open {
    /* Styling when the dropdown is open */
    border: 1px solid #666;
}

QComboBox::indicator {
    /* Styling for the selection indicator (the small arrow on the right) */
    border: none;
}
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="submit_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>170</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 700 15pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
